gnomad_qc/v3/resources/basics.py

```diff
@@ -82,7 +82,7 @@ def get_gnomad_v3_mt(
 
 def qc_temp_prefix(version: str = CURRENT_RELEASE) -> str:
     """
-    Returns path to temporary QC bucket.
+    Return path to temporary QC bucket.
```

Comment:
Thanks for adding these! They were put in before we synced up on this across the group